### PR TITLE
Story/quad 209

### PR DESCRIPTION
**Pull request by yyk16293**

### CODE REVIEW

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/resolver/EditResolverController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    @RequestMapping(value = "/auth/resolvers/update", method = RequestMethod.POST)
+    public String editResolver(Principal principal,
+            @Validated @ModelAttribute("resolver") ProjectHandleResolver resolver, BindingResult results, Model model) {
+
+        if (results.hasErrors()) {
+            model.addAttribute("resolver", resolver);
+            return "auth/resolvers/edit";
+        }
+
+        resolver.setUsername(principal.getName());
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/resolver/EditResolverController.java]: why do you set the username here?

Comment (yyk16293) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/resolver/EditResolverController.java]: As it's value was null, I was setting it's value here. I made changes to the code, now it's being handled using hidden value in the form being submitted.

Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/resolver/EditResolverController.java]: If you submit a hidden field with the correct field name (the one it has in the bean), it'll be set automatically.

Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/resolver/EditResolverController.java]: never mind. Now I see that you changed it.

**Comment by jdamerow** [comment]
ok to test

**Pull request merged**